### PR TITLE
🚩 Deploys

### PR DESCRIPTION
**Pull request by micdijkstra**
This PR will:

- Add ability to create, list and delete deploys for a site
- Update list snapshots to use the paginated schema

Will be released as part of #49 

### Todo

- [x] Show command for next page (deploys and snapshots)
- [x] Update examples

### CODE REVIEW

File: src/cli/site/deploys.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      maxLineWidth: 'auto'
+    })
+  )
+
+  console.log(
+    '\n',
+    columnify(
+      [
+        {
+          'More?': index.pageInfo.hasNextPage,
```
Comment (benschwarz) [inline src/cli/site/deploys.js]: Is this just for debugging/development?

File: src/utils/cli.js [hunk 2]
```diff
@@ -6,7 +6,14 @@ const options = {
     requiresArg: true
   },
   json: { describe: 'Return the output in JSON format' },
-  csv: { describe: 'Return the output in CSV format' }
+  csv: { describe: 'Return the output in CSV format' },
+  count: {
+    default: 25,
+    describe: 'The number of records to return'
```
Comment (benschwarz) [inline src/utils/cli.js]: Is there a maximum?

File: src/cli/site/deploys.js (hunk 1 above)
Comment (benschwarz) [reply src/cli/site/deploys.js]: Perhaps we could figure out the command that they need to run in order to get the next page, then supply that? 

eg: 

> To see deploys prior to release `djd301`, run `calibre site deploy-list --site=calibre --cursor="M2o="`

Comment (micdijkstra) [reply src/cli/site/deploys.js]: No, this outputs if there's more pages and what the cursor is. e.g.

```
MORE? | CURSOR
true  | asd234jkln
```

It does feel a little clunky …

**Comment by benschwarz** [comment]
What are the commands that people will run? Are they consistent with other list/show commands?

**Comment by benschwarz** [comment]
Could you add to the `examples/nodejs` directory as part of this?

File: src/cli/site/deploys.js (hunk 1 above)
Comment (micdijkstra) [reply src/cli/site/deploys.js]: Yeah, I just had the same thought! Will update.

File: src/utils/cli.js (hunk 2 above)
Comment (micdijkstra) [reply src/utils/cli.js]: Yep good pick-up, it's 500. Will add to the message.

**Comment by micdijkstra** [comment]
> What are the commands that people will run? Are they consistent with other list/show commands?

The commands are identical to snapshots.

**Pull request merged**